Framework/PythonInterface/test/python/plugins/functions/RedfieldTest.py

```diff
@@ -18,18 +18,16 @@ def test_function_has_been_registered(self):
 
     def test_function_output(self):
         input = np.linspace(0, 1000, 4)  # Between 0-1000 Gauss
-        expected = [0.91847597, 0.76278678, 0.5056509, 0.3237545]
+        expected = [0.091847597, 0.076278678, 0.05056509, 0.03237545]
         tolerance = 1.0e-05
-        function_params = {"A0": 10, "Hloc": 50, "Tau": 0.1}
```

Comment:
since we had A0*H_loc^2, removing A0 lead to the expected results being divided by 10

Reply:
Yes, this is why I've updated the expected results